socorro/siglists/README.md

```diff
@@ -1,3 +1,62 @@
 # Signatures Utilities Lists
 
-Great documentation comes here.
+This folder contains lists that are used to configure the C signature generation process. Each ``.txt`` file contains a list of signatures or regex matching signatures, that are used at various steps of our algorithm.
+
+## Signature Generation Algorithm
+
+When generating a C signature, 4 steps are involved.
+
+1. We walk the crashing thread's stack, looking for things that would match the [signature sentinels](#signature-sentinels). The first matching element, if any, becomes the top of the sub-stack we'll consider going forward.
```

Comment:
Perhaps "signature sentinels" should be written as "Signature Sentinels" so it matches the header that comes a few lines further down.